tests/unit/test_vcs.py

```diff
@@ -379,12 +379,32 @@ def test_get_git_version():
     assert git_version >= parse_version('1.0.0')
 
 
+@pytest.mark.parametrize('isatty_return,use_interactive,expected', [
```

Comment:
This is a tiny suggestion, but I think the test cases would be a little easier to understand if `use_interactive` occurs in the argument list before `isatty_return`, because `use_interactive` is the more important argument and takes precedence (and `isatty_return` doesn't necessarily come into play).